webrtc-stats.html

```diff
@@ -1400,6 +1404,56 @@
                   applied. Calculated in decibels, as defined in [[!ECHO]] (2012) section 3.15.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>totalSamples</code></dfn> of type <span class="idlMemberType"><a>unsigned
+                long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only present for inbound audio tracks. The total number of audio samples that have
+                  been received for this track. This includes <a>concealedSamples</a>.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>concealedSamples</code></dfn> of type <span class="idlMemberType"><a>
+                unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only present for inbound audio tracks. The total number of audio samples that are
+                  concealed samples. A concealed sample is a sample that has been synthesized to
+                  conceal packet loss.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>concealedAudibleSamples</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only present for inbound audio tracks. The total number of concealed audio samples
+                  during audible portions of the stream. Audible means that the received audio is
```

Comment:
No explanation of what "concealed" means here.
I think it should say "Number of samples that were played out based on synthesized data and not based on incoming data".

This shouldn't include comfort noise played out when CN is used, I think.